src/Kovan-DssSpell.sol

```diff
@@ -44,94 +75,218 @@ contract DssSpellAction is DssAction {
+        require(ClipAbstract(col.clipper).ilk() == col.ilk, "DssSpell/clip-wrong-ilk");
+        // Set CLIP for ETH-A in the DOG
+        DssExecLib.setContract(col.dog, col.ilk, "clip", col.clipper);
+        // Set VOW in the ETH-A CLIP
+        DssExecLib.setContract(col.clipper, "vow", col.vow);
+        // Set CALC in the ETH-A CLIP
+        DssExecLib.setContract(col.clipper, "calc", col.calc);
         // Authorize CLIP can access to VAT
-        DssExecLib.authorize(MCD_VAT, MCD_CLIP_YFI_A);
-
+        DssExecLib.authorize(col.vat, col.clipper);
         // Authorize CLIP can access to DOG
-        DssExecLib.authorize(MCD_DOG, MCD_CLIP_YFI_A);
-
+        DssExecLib.authorize(col.dog, col.clipper);
         // Authorize DOG can kick auctions on CLIP
-        DssExecLib.authorize(MCD_CLIP_YFI_A, MCD_DOG);
-
-        // Authorize the new END to access the YFI CLIP
-        DssExecLib.authorize(MCD_CLIP_YFI_A, MCD_END);
-
+        DssExecLib.authorize(col.clipper, col.dog);
+        // Authorize the new END to access the ETH CLIP
+        DssExecLib.authorize(col.clipper, col.end);
         // Authorize CLIPPERMOM can set the stopped flag in CLIP
-        DssExecLib.authorize(MCD_CLIP_YFI_A, CLIPPER_MOM);
-
-        // Authorize new ESM to execute in YFI-A Clipper
-        DssExecLib.authorize(MCD_CLIP_YFI_A, MCD_ESM);
-
-        // Whitelist CLIP in the YFI osm
-        DssExecLib.addReaderToOSMWhitelist(PIP_YFI, MCD_CLIP_YFI_A);
-
-        // Whitelist CLIPPER_MOM in the YFI osm
-        DssExecLib.addReaderToOSMWhitelist(PIP_YFI, CLIPPER_MOM);
-
+        DssExecLib.authorize(col.clipper, col.clipperMom);
+        // Authorize new ESM to execute in ETH-A Clipper
+        DssExecLib.authorize(col.clipper, col.esm);
+        // Whitelist CLIP in the ETH osm
+        DssExecLib.addReaderToOSMWhitelist(col.pip, col.clipper);
+        // Whitelist clipperMom in the ETH osm
+        DssExecLib.addReaderToOSMWhitelist(col.pip, col.clipperMom);
         // No more auctions kicked via the CAT:
-        DssExecLib.deauthorize(MCD_FLIP_YFI_A, MCD_CAT);
-
-        // No more circuit breaker for the FLIP in YFI-A:
-        DssExecLib.deauthorize(MCD_FLIP_YFI_A, DssExecLib.flipperMom());
-
-        Fileable(MCD_DOG).file("YFI-A", "hole", 5_000 * RAD);
-        Fileable(MCD_DOG).file("YFI-A", "chop", 113 * WAD / 100);
-        Fileable(MCD_CLIP_YFI_A).file("buf", 130 * RAY / 100);
-        Fileable(MCD_CLIP_YFI_A).file("tail", 140 minutes);
-        Fileable(MCD_CLIP_YFI_A).file("cusp", 40 * RAY / 100);
-        Fileable(MCD_CLIP_YFI_A).file("chip", 1 * WAD / 1000);
-        Fileable(MCD_CLIP_YFI_A).file("tip", 0);
-        Fileable(MCD_CLIP_CALC_YFI_A).file("cut", 99 * RAY / 100); // 1% cut
-        Fileable(MCD_CLIP_CALC_YFI_A).file("step", 90 seconds);
-
+        DssExecLib.deauthorize(col.flipper, col.cat);
+        // No more circuit breaker for the FLIP in ETH-A:
+        DssExecLib.deauthorize(col.flipper, col.flipperMom);
```

Comment:
Are we sure we want to do this? If we ever go back to the Flipper, we have to remember to re-authorize the FlipperMom, and I don't see too much harm in leaving it authorized.

OTOH, the current large gas refund is nice and there's an argument that deauthorizing while that's still available makes sense as the probability of going back is hopefully low.

Reply:
We decided to do this so governance can't fallback to the flipper without pause delay.